ledger/src/entry.rs

```diff
@@ -255,7 +316,7 @@ impl EntrySlice for [Entry] {
             }
         });
 
-        let tx_hashes: Vec<Option<Hash>> = PAR_THREAD_POOL.with(|thread_pool| {
+        let tx_hashes = PAR_THREAD_POOL.with(|thread_pool| {
```

Comment:
we could also move this block of logic into the gpu_verify_thread so that replay can start earlier.

Reply:
Ok, but not sure it matters if you start it before or after and this is also pretty cheap.